static/i18n/en/app.json

```diff
@@ -1246,6 +1256,14 @@
         "2": "Press both buttons three times to access the Dashboard.",
         "3": "Open the Manager and click Update firmware."
       }
+    },
+    "UnavailableTezosOriginatedAccountReceive": {
+      "title": "You can't receive funds on originated account",
+      "description": "If you want to receive funds, please use the parent account"
+    },
+    "UnavailableTezosOriginatedAccountSend": {
+      "title": "Sorry, you can't send funds from originated account yet",
+      "description": "Please contact Ledger Support for more information"
     }
```

Comment:
"title": "Cannot send from subaccounts yet",
"description": "This feature will be added at a later stage due to changes recently introduced by the Babylon amendment."